src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -293,14 +293,25 @@ uiModules
 
   function transferVisState(fromVis, toVis, stage) {
     return function () {
+
+      //verify this before we copy the "new" state
+      const aggregationChanges = !fromVis.aggs.jsonDataEquals(toVis.aggs);
```

Comment:
This might be clearer if we prefix with `is`, `has`, or `should` to indicate the boolean nature of the var. How about `isAggregationsChanged`?